src/app/shared/stache/modules/image/image.component.html

```diff
@@ -0,0 +1 @@
+<img class="stache-image" [src]="imageSource" />
```

Comment:
Let's wrap this in a `<div class="stache-image"><img /></div>` so that we have more styling flexibility in the future.